src/components/PlaylistOverview.vue

```diff
@@ -1,48 +1,41 @@
 <script setup lang="ts">
 import filters from "@/utils/filters";
 import { ref, Ref } from "vue";
-import { PlaylistModel } from "@bcc-code/bmm-sdk-fetch";
-
-import { list } from "@/api/playlists";
-import { useAuth0 } from "@auth0/auth0-vue";
+import {
+  PlaylistApi,
+  PlaylistModel,
+  Configuration,
+} from "@bcc-code/bmm-sdk-fetch";
 
 const playlists: Ref<PlaylistModel[]> = ref([]);
 
-const { getAccessTokenSilently } = useAuth0() ?? {
```

Comment:
I think a merge went bad since you're undoing a lot of changes from other people

Reply:
The commit has been fixed ! Please can have a look again ? Thanks